### PR TITLE
Create R-project.xml

### PR DESCRIPTION
**Pull request by davidabian**

### CODE REVIEW

**Comment by Bisaloo** [comment]
Thank you for this PR. I was planning to do it at some point.

Could you add rules for:

- lists.r-project.org
- wiki.r-project.org
- r-forge.r-project.org
- beta.r-project.org

and comments for:

- ess.r-project.org (mismatched)
- latticeextra.r-forge.r-project.org (mismatched)
- translation.r-project.org
- www.cran.r-project.org (mismatched) (not the same as cran.r-project.org)
- search.r-project.org
- rhrv.r-forge.r-project.org (mismatched)
- win-builder.r-project.org
- debian.cran.r-project.org (timeout)

**Comment by davidabian** [comment]
You're welcome!

* lists.r-project.org shows wrong contents via HTTPS.
* wiki.r-project.org shows wrong contents via HTTPS (and a broken redirect via HTTP).
* beta.r-project.org shows wrong contents via HTTPS.

So I would add a rule for r-forge.r-project.org, and comments for the rest. Do you agree?

**Comment by Bisaloo** [comment]
Sure. Thank you!

**Comment by J0WI** [comment]
Thanks!
Just one more (mismatched): `cran.us.r-project.org`

File: src/chrome/content/rules/R-project.xml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+		* latticeextra.r-forge.r-project.org
+		* lists.r-project.org
+		* rhrv.r-forge.r-project.org
+		* search.r-project.org
+		* translation.r-project.org
+		* wiki.r-project.org
+		* win-builder.r-project.org
+		* www.cran.r-project.org
+
+--><ruleset name="R Project">
```
Comment (J0WI) [inline src/chrome/content/rules/R-project.xml]: please add a line break between these tags.

**Comment by davidabian** [comment]
Done.

**Comment by davidabian** [comment]
@J0WI: Something not related to this pull request went wrong.
```
failure: /opt/utils/../src/chrome/content/rules/Washington-Post-Company.xml failed test: The 'from' rule contains unescaped period in regular expression.  Try escaping it with a backslash.
```

**Comment by J0WI** [comment]
Oh, this was me in #4930.
I wonder why the tests passed there.
I pushed a hotfix and will rerun the test.

**Pull request merged**